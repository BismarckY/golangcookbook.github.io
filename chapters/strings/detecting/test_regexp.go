package main

import (
	"fmt"
	"regexp"
)

func main() {
	matched, err := regexp.MatchString("Wor.*", "Hello World")
	fmt.Println("Matched:", matched, "Error:", err)
	matched, err = regexp.MatchString("Good.*", "Hello World")
	fmt.Println("Matched:", matched, "Error:", err)
	matched, err = regexp.MatchString("(bad regexp", "Hello World")
	fmt.Println("Matched:", matched, "Error:", err)
}
