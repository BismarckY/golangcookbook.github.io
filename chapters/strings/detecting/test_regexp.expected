Matched: true Error: <nil>
Matched: false Error: <nil>
Matched: false Error: error parsing regexp: missing closing ): `(bad regexp`
